services/core-api/app/api/now_applications/models/now_application.py

```diff
@@ -21,30 +23,46 @@ class NOWApplication(Base, AuditMixin):
     _edit_groups = [NOW_APPLICATION_EDIT_GROUP]
     _edit_key = NOW_APPLICATION_EDIT_GROUP
 
-    now_application_id = db.Column(db.Integer, primary_key=True, server_default=FetchedValue())
-    now_application_identity = db.relationship(
-        'NOWApplicationIdentity', lazy='selectin', uselist=False)
-    now_application_guid = association_proxy('now_application_identity', 'now_application_guid')
+    now_application_id = db.Column(db.Integer,
+                                   primary_key=True,
+                                   server_default=FetchedValue())
+    now_application_identity = db.relationship('NOWApplicationIdentity',
+                                               lazy='selectin',
+                                               uselist=False)
+    now_application_guid = association_proxy('now_application_identity',
+                                             'now_application_guid')
 
     mine_guid = association_proxy('now_application_identity', 'mine_guid')
     mine_name = association_proxy('now_application_identity', 'mine.mine_name')
     mine_no = association_proxy('now_application_identity', 'mine.mine_no')
-    mine_region = association_proxy('now_application_identity', 'mine.mine_region')
+    mine_region = association_proxy('now_application_identity',
+                                    'mine.mine_region')
     now_number = association_proxy('now_application_identity', 'now_number')
 
-    lead_inspector_party_guid = db.Column(UUID(as_uuid=True), db.ForeignKey('party.party_guid'))
+    lead_inspector_party_guid = db.Column(UUID(as_uuid=True),
+                                          db.ForeignKey('party.party_guid'))
     lead_inspector = db.relationship('Party', lazy='selectin', uselist=False)
 
     now_tracking_number = db.Column(db.Integer)
     notice_of_work_type_code = db.Column(
-        db.String, db.ForeignKey('notice_of_work_type.notice_of_work_type_code'), nullable=False)
+        db.String,
+        db.ForeignKey('notice_of_work_type.notice_of_work_type_code'),
+        nullable=False)
     notice_of_work_type = db.relationship('NOWApplicationType', lazy='joined')
 
     now_application_status_code = db.Column(
         db.String,
         db.ForeignKey('now_application_status.now_application_status_code'),
         nullable=False)
-    status_updated_date = db.Column(db.Date, nullable=False, server_default=FetchedValue())
+    status_updated_date = db.Column(db.Date,
+                                    nullable=False,
+                                    server_default=FetchedValue())
+    last_updated_date = db.Column(db.Date,
+                                  default=datetime.utcnow,
```

Comment:
I don't think `last_updated_date` and `last_updated_by` should have default values and they should be nullable because NoW's can exist without having been updated.